### PR TITLE
exercises: MatchImage

### LINKED ISSUE #21: exercise: "Markieren Sie das richtige Bild"

### PR DESCRIPTION
**Pull request by inyono**
Close #21

### CODE REVIEW

**Comment by inyono** [comment]
depends on #42

File: src/components/Components.js [hunk 1]
```diff
@@ -3,22 +3,30 @@ import { TouchableOpacity, View, Image, Text } from 'react-native';
 
 const styles = {
   buttonWithInset: {
```
Comment (Knorrke) [inline src/components/Components.js]: unused styles

**Comment by inyono** [comment]
Yeah, you are right. I wanted to do the "correct" logic etc. later when we write a HOC for this. But it should at least have the selected styles. Will add that.

**Comment by inyono** [comment]
I will wait for #44 before fixing this since I don't want to duplicate the `Components` refactoring

**Comment by inyono** [comment]
@Knorrke Ready for review again

**Comment by Knorrke** [comment]
CI check failing seems to be false, since the tests passed in the other build, but you might want to check that for yourself.

**Comment by inyono** [comment]
Yeah, weird. But should be fine

**Pull request merged**